app/enterprise/1.5.x/kong-for-kubernetes/deployment-options.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Kong for Kubernetes deployment options
+---
+
+- [Deployment options](#deployment-options)
+- [DB-less versus database-backed deployments](#db-less-versus-database-backed-deployments)
+- [Choosing between DB-less or database-backed deployments](#choosing-between-db-less-or-database-backed-deployments)
+  * [Feature availability](#feature-availability)
+  * [Plugin compatibility](#plugin-compatibility)
+  * [Manual configuration](#manual-configuration)
+- [Migrating between deployment types](#migrating-between-deployment-types)
+
+## Deployment options
+
+Kong for Kubernetes provides two options for its Kong Enterprise container
+image:
+
+* [kong-enterprise-k8s][k8s-bintray] (Kong for Kubernetes Enterprise)
+* [kong-enterprise-edition][enterprise-bintray] (Kong for Kubernetes with Kong
+  Enterprise)
+
+_These repositories require a login. If you see a 404, log in through the [Kong
+repository home page](https://bintray.com/kong) first._
+
+The `kong-enterprise-k8s` image is recommended for most deployments. It
+provides most Kong Enterprise plugins and runs without a database, but does not
+provide other Kong Enterprise features (Kong Manager, Dev Portal, Vitals,
+etc.).
+
+The `kong-enterprise-edition` image is recommended for deployments that require
+features not supported by `kong-enterprise-k8s`. It supports all Kong
+Enterprise plugins and features, but cannot run without a database.
+
+## DB-less versus database-backed deployments
+
+When using Kong for Kubernetes, the source of truth for Kong's configuration is
+the Kubernetes configuration in etcd: Kong's custom Kubernetes resources,
+ingresses, and services provide the information necessary for the ingress
+controller to configure Kong. This differs from Kong deployments that do not
+use an ingress controller, where configuration in the database or DB-less
+config file is the source of truth.
+
+In traditional deployments, Kong's database (PostgreSQL or Cassandra) provides
+a persistent store of configuration available to all Kong nodes to ensure
+consistent proxy behavior across the cluster that is not affected by node
+restarts. Because etcd provides this functionality in Kong for Kubernetes
+deployments, it is not necessary to run an additional database, reducing
+maintenance and infrastructure requirements.
+
+While Kong for Kubernetes does not require a database, it is fully compatible
+with PostgreSQL and requires it for some features. etcd remains the source of
+truth, but the controller can translate Kubernetes resources from etcd into either
+DB-less or database configuration.
```

Comment:
Can we skip DB-less here? This cause confusion and I had to read this section 3 times to figure out what we are talking about.
The previous paragraph has enough explanation for DB-less and we should focus on database-mode in this one. 
We should explain here that in this case, the controller will configure Kong's database to match what is stored inside etcd.